Fix a compilation problem on Solaris due to obsolete patch-ac.

--- tun.c.orig	2008-10-09 09:06:50.000000000 +0000
+++ tun.c
@@ -721,12 +721,13 @@
 			    );
 	}
       else
-	openvpn_snprintf (command_line, sizeof (command_line),
-			  IFCONFIG_PATH " %s %s netmask %s broadcast + up",
-			  actual,
-			  ifconfig_local,
-			  ifconfig_remote_netmask
-			  );
+	  argv_printf (&argv,
+			    "%s %s netmask %s broadcast + up",
+			    IFCONFIG_PATH,
+			    actual,
+			    ifconfig_local,
+			    ifconfig_remote_netmask
+			    );
 
       argv_msg (M_INFO, &argv);
       if (!openvpn_execve_check (&argv, es, 0, "Solaris ifconfig phase-2 failed"))
